modin/core/io/text/text_file_dispatcher.py

```diff
@@ -597,18 +597,14 @@ def _launch_tasks(cls, splits: list, **partition_kwargs) -> Tuple[list, list, li
         dtypes_ids : list
             array with references to the partitions dtypes objects.
         """
-        partition_ids = []
-        index_ids = []
-        dtypes_ids = []
-        for start, end in splits:
+        partition_ids = [None] * len(splits)
+        index_ids = [None] * len(splits)
+        dtypes_ids = [None] * len(splits)
+        for idx, (start, end) in enumerate(splits):
             partition_kwargs.update({"start": start, "end": end})
-            partition_id = cls.deploy(
+            *partition_ids[idx], index_ids[idx], dtypes_ids[idx] = cls.deploy(
```

Comment:
you **did** verify this works, right?.. there are tests which pass and cover this line, right?

Reply:
Yes, I am verified. All csv tests go through this place.

Reply:
I will split the changes into another PR (refactoring)